code-of-conduct.md

```diff
@@ -1,3 +0,0 @@
-# Kubernetes Community Code of Conduct
```

Comment:
Was this removal unintentional?

Reply:
yup. errant `rm *.md` in the root then not restoring.